Formula/kfctl.rb

```diff
@@ -0,0 +1,28 @@
+class Kfctl < Formula
+  desc "Kubeflow command-line interface"
+  homepage "https://github.com/kubeflow/kfctl"
+  url "https://github.com/kubeflow/kfctl.git",
+      tag: "v1.1.0",
+      revision: "9a3621e2b2ce6d2baa5c78acccec5028c4e4cbe1"
```

Comment:
```suggestion
  url "https://github.com/kubeflow/kfctl.git",
      tag:      "v1.1.0",
      revision: "9a3621e2b2ce6d2baa5c78acccec5028c4e4cbe1"
```

Reply:
This should align these

Reply:
Took care of it using 
```
brew audit --strict --online --fix kfctl
kfctl:
  * 2: col 18: [Corrected] Description should use "command-line" instead of "commandline".
  * 23: col 1: [Corrected] Trailing whitespace detected.
Error: 2 problems in 1 formula detected, 2 problems corrected
```